src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,231 +45,32 @@ const NavBar: React.FC<NavBarPropsType> = ({
-                        <Popover.Content>
-                          <RadioModal
-                            {...{
-                              volume,
-                              setVolume,
-                              title: venue?.radioTitle,
-                            }}
-                            onEnableHandler={handleRadioEnable}
-                            isRadioPlaying={isRadioPlaying}
-                          />
-                        </Popover.Content>
-                      </Popover>
-                    }
-                    rootClose={true}
-                    defaultShow={showRadioOverlay}
-                  >
-                    <div
-                      className={`profile-icon navbar-link-radio ${
-                        volume === 0 && "off"
-                      }`}
-                    />
-                  </OverlayTrigger>
-                )}
-
-                {showSoundCloudRadio && (
-                  <S.RadioTrigger>
-                    <div
-                      className={`profile-icon navbar-link-radio ${
-                        volume === 0 && "off"
-                      }`}
-                      onClick={toggleShowRadioPopover}
-                    />
-
-                    <S.RadioWrapper showRadioPopover={showRadioPopover}>
-                      <iframe
-                        title="venueRadio"
-                        id="sound-cloud-player"
-                        scrolling="no"
-                        allow="autoplay"
-                        src={`https://w.soundcloud.com/player/?url=${radioStation}&amp;start_track=0&amp;single_active=true&amp;show_artwork=false`}
-                      />
-                    </S.RadioWrapper>
-                  </S.RadioTrigger>
-                )}
-
-                <OverlayTrigger
-                  trigger="click"
-                  placement="bottom-end"
-                  overlay={ProfilePopover}
-                  rootClose={true}
-                >
-                  <div className="navbar-link-profile">
-                    <img
-                      src={profileImage}
-                      className="profile-icon"
-                      alt="avatar"
-                      width="40"
-                      height="40"
-                    />
-                  </div>
-                </OverlayTrigger>
+                {rightSlot}
+                <NavBarProfile />
               </div>
+            ) : (
+              <NavBarLogin />
             )}
           </div>
         </div>
       </header>
 
-      <div
-        className={`schedule-dropdown-backdrop ${
-          isEventScheduleVisible ? "show" : ""
-        }`}
-        onClick={hideEventSchedule}
-      >
-        <SchedulePageModal isVisible={isEventScheduleVisible} />
-      </div>
-
       {/* @debt Remove back button from Navbar */}
-      {hasBackButton && venue?.parentId && parentVenue?.name && (
+      {venue?.parentId && parentVenue?.name && (
```

Comment:
You have `parentVenueId` extracted at the top of this component. Worth reusing it here?